place_tracker/lib/place_map.dart

```diff
@@ -489,9 +489,9 @@ class _CategoryButtonBar extends StatelessWidget {
         child: ButtonBar(
           alignment: MainAxisAlignment.center,
           children: [
-            ElevatedButton(
-              style: ElevatedButton.styleFrom(
-                  foregroundColor:
+            FilledButton(
+              style: FilledButton.styleFrom(
+                  backgroundColor:
```

Comment:
Previous code set the `foregroundColor` instead, which did nothing because the `Text` color was set to white anyway. I guess the original intention was to change the `backgroundColor` instead.

Also switched to FilledButton instead of ElevatedButton.